packages/server/tests/AuthenticationSignUp.test.ts

```diff
@@ -0,0 +1,31 @@
+import Database from '@ioc:Adonis/Lucid/Database';
+import { SignUpResponseBody } from '@musicroom/types';
+import User from 'App/Models/User';
+import test from 'japa';
+import supertest from 'supertest';
+import urlcat from 'urlcat';
+import { BASE_URL, TEST_AUTHENTICATION_GROUP_PREFIX } from './utils/TestUtils';
+
+test.group('Users Profile information tests', (group) => {
+    group.beforeEach(async () => {
+        await Database.beginGlobalTransaction();
+    });
+
+    group.afterEach(async () => {
+        await Database.rollbackGlobalTransaction();
+    });
+
+    test('It should unfollow given user', async (assert) => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏The name of the test is not accurate.